lib/videoroom/notifier.ex

```diff
@@ -12,14 +12,8 @@ defmodule Videoroom.Notifier do
   @impl true
   def init(opts) do
     Supervisor.start_link([{Jellyfish.Notifier, opts}], strategy: :one_for_one)
+    Logger.info("Successfully connected")
     Process.monitor(Jellyfish.Notifier)
-
-    {:ok, nil}
-  end
-
-  @impl true
-  def handle_info({:DOWN, _ref, :process, _object, _reason}, state) do
-    Logger.warning("Jellyfish Notifier exited")
-    {:noreply, state}
+    {:ok, %{}}
   end
```

Comment:
Why has this been removed? If Jellyfish stops working for some reason and the Notifier connection is closed is there any way to acknowledge it?

Reply:
I think we could acknowledge it in the Meeting process.

Reply:
Ok, could you please add the monitoring and logging of the notifier in the meeting?